.github/workflows/linting.yml

```diff
@@ -5,6 +5,17 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 jobs:
+  internal_investigation:
+    runs-on: ubuntu-latest
+    name: Internal Investigation
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ruby # Latest stable CRuby version
+          bundler-cache: true
+      - run: NO_COVERAGE=true bundle exec rake internal_investigation
```

Comment:
Internal investigation shouldn't be bothered by NO_COVERAGE, or why is it?

Reply:
You're right, I had this from the main job where I extracted this from. Will remove